packages/storybook/src/nimble/table/table.mdx

```diff
@@ -97,6 +97,69 @@ The expected usage of the dynamically loaded hierarchy is as follows:
 -   calling `setData` will clear options associated with IDs that are no longer
     present in the data
 
-{/* ## Accessibility */}
+## Accessibility
+
+### Keyboard Navigation
+
+Once the table is focused, it supports navigation via keyboard keys.
+
+-   If the table is focused via `Tab`/`Shift-Tab`: focus starts on the first
+    action button in the header row (if any); the first column header (if any
```

Comment:
<nit> Not sure if using semi-colons is appropriate here as the separated sets of text are not complete sentences. I think commas will do.

Reply:
Updated